src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -390,7 +390,7 @@ internal string CreateSearchQuery(string[] fields, string SearchTerm, bool IsPac
                     foreach (string s in searchTerm.Split(' '))
                     {
                         //If is a ByEmptySpace search and the split words match with more than MaxNodeNamesRepeated nodes then the word is skipped (otherwise the results will be polluted with hundred of not related nodes)
-                        int nodesFrequency = dynamoModel.SearchModel.Entries.Where(entry => entry.Name.ToLower().Contains(s) && !string.IsNullOrEmpty(s)).Count();
+                        int? nodesFrequency = dynamoModel.SearchModel?.Entries.Where(entry => entry.Name.ToLower().Contains(s) && !string.IsNullOrEmpty(s)).Count();
                         if (nodesFrequency > MaxNodeNamesRepeated) continue;
```

Comment:
wont this throw an exception if nodesFrequency is null ?

Reply:
no because nodesFreq is nullable

![image](https://github.com/user-attachments/assets/182c1154-7b93-4ad0-8a37-4e4d4752836b)